### ISSUE TITLE
Notification web_hook payload transforms "<" to unicode characters

### ISSUE THREAD

**Issue by estif95** [dynatrace-oss/terraform-provider-dynatrace#254]
**Describe the bug**
on dynatrace_notification resource, payload of nested_schema web_hook transforms "<" to unicode characters
**To Reproduce**
1. Create a dynatrace_notification resource of web_hook type and use "<" on the payload.
2. Go to dynatrace website to find the new created resource and will find "\u003c" on the message notification

**Expected behavior**
1. Create a dynatrace_notification resource of web_hook type and use "<" on the payload.
2. Go to dynatrace website to find the new created resource and will find "<" on the message notification

**Additional context**
Changing value via website works as expected with "<"

Thanks in advance!

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hello @estif95, I am unable to reproduce the issue you're describing. We haven't made any changes in resource `dynatrace_notification` for months. Therefore it's likely that this has been an issue on the Dynatrace REST API side that has been resolved recently.
What version of Dynatrace are you using Terraform with? I am currently testing against v1.266.116.
The resource I am testing with looks like this:
```
resource "dynatrace_notification" "dynatrace_webhook_notification" {
  web_hook {
    name                   = "tf_dynatrace_webhook_notification"
    accept_any_certificate = true
    active                 = false
    alerting_profile       = "c21f969b-5f03-333d-83e0-4f8f136e7682"
    notify_event_merges    = true
    payload                = "web-hook-payload < web-hook-payload"
    url                    = "https://webhook.site/40bf4d43-1a50-4ebd-913d-bf50ce7c3a0a"
    header {
      name  = "http-header-name-01"
      value = "http-header-value-01"
    }
    header {
      name  = "http-header-name-02"
      value = "http-header-value-02"
    }
  }
}
```

**Comment by estif95** [comment]
Hello, thanks for reviewing and answering the issue! Reviewing the problem again it seems a Terraform behaviour with jsonencode function and unicode characters. I managed to deploy your snippet code and it works as expected. We were sending a complex json payload with multiple double quotes and single quotes "" ''. Closing the issue as is not related with dynatrace provider.

Thanks again and sorry for inconveniences!
